.github/workflows/ci-test.yml

```
@@ -0,0 +1,50 @@
+name: CI-Test
+
+on:
+  push:
+    branches: 
```

yamllint

[error] trailing spaces

(trailing-spaces)